### PR TITLE
tf-14 upgrade

### PR DESCRIPTION
**Pull request by Nuru**
## what
- Test of tf-14 upgrade

## why
- Support tf-14

### CODE REVIEW

File: examples/complete/versions.tf [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 
   required_providers {
-    local = "~> 1.2"
+    local = {
+      source  = "hashicorp/local"
+      version = ">= 1.2"
+    }
+    aws = {
+      source = "hashicorp/aws"
+    }
```
Comment (aknysh) [inline examples/complete/versions.tf:11]: version = ">= 2"

**Review by aknysh — DISMISSED** [review]

**Review by osterman — APPROVED** [review]

**Pull request merged**